assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],
+        "recipeFluids": ["0.25*Fluid:Water"],

```

Comment:
It isn't mandatory for a recipe to include an amount of water. For toast it probably makes more sense to leave this requirement out :-)

Down the road we should be able to add a utensil requirement, then instead we could have toast require a knife or something to slice the bread. I think that would make sense.